bot/exts/info/doc/_cog.py

```diff
@@ -42,6 +44,10 @@
 
 COMMAND_LOCK_SINGLETON = "inventory refresh"
 
+REDO_EMOJI = "\U0001f501"  # :repeat:
+REDO_TIMEOUT = 30
+COMPACT_DOC_REGEX = r"\[\[(?P<symbol>\S*)\]\]"
```

Comment:
The reason I suggested matching specifically 
```py
[[`item`]]
```
is that this won't match Python lists like `[[]]` or `[[initial]]` or `[[1,2,3]]` and lead to confusing/noisy messages from the bot. Maybe this could be changed to
```py
r"\[\[`(?P<symbol>[^\s\]`]*)`\]\]"
```
? This would also remove the need to strip \` from matches